src/__tests__/safes.test.ts

```diff
@@ -1,10 +1,19 @@
 import SafeL2141 from './assets/v1/v1.4.1/safe_l2.json';
+import SafeL2141v2 from '../assets/v1.4.1/safe_l2.json';
 import Safe141 from './assets/v1/v1.4.1/safe.json';
+import Safe141v2 from '../assets/v1.4.1/safe.json';
 import GnosisSafe130 from './assets/v1/v1.3.0/gnosis_safe.json';
 import GnosisSafe120 from './assets/v1/v1.2.0/gnosis_safe.json';
 import GnosisSafe111 from './assets/v1/v1.1.1/gnosis_safe.json';
 import GnosisSafe100 from './assets/v1/v1.0.0/gnosis_safe.json';
-import { getSafeSingletonDeployment, getSafeL2SingletonDeployment } from '../safes';
+import {
+  getSafeSingletonDeployment,
+  getSafeL2SingletonDeployment,
+  getSafeSingletonDeployments,
+  getSafeL2SingletonDeployments,
+} from '../safes';
+// @ts-expect-error The function is conditionally exported only if the process.env.NODE_ENV is 'test'. TypeScript will not be able to figure it out.
+import { mapJsonToDeploymentsFormatV2 } from '../utils';
```

Comment:
uber-nit: To abstract this away, you could have a `src/__tests__/utils.ts` file that re-exports this symbol, so you only have to work around and document this TS issue in a single plance, and all tests import from:

```suggestion
import { mapJsonToDeploymentsFormatV2 } from './utils';
```